### PR TITLE
iOS bug fix

### LINKED ISSUE #27: [Ionic 4] Can't get it to work for iOS app.
The plugin doesn't seem to work for iOS and I know the plugin isn't tested on iOS. 
What is happening for me is, the app crashes but the crash reports are NOT submitted to the Firebase. 
I followed all the steps mentioned and the app even crashes successfully (Test crash), but I don't see the communication happening between Firebase and my app. I don't see the Crashlytics dashboard for iOS app and instead I'm stuck at -- "Build and run your app". 

Has anybody got this plugin work for Ionic iOS app? 
Please let me know.
Thanks.

### PR DESCRIPTION
**Pull request by Darcelsan**
Fixes #27

## Proposed Changes

  - Add initialization required in iOS
  - Add Firebase imports required by the Firebase documentation

### CODE REVIEW

File: plugin.xml [hunk 1]
```diff
@@ -74,6 +74,9 @@
         <pod name="Fabric"/>
         <pod name="Crashlytics" />
         <pod name="FirebaseCore" />
+        <pod name="Firebase/Analytics" />
+        <pod name="Firebase/Auth" />
+        <pod name="Firebase/Firestore" />
```
Comment (piotr-cz) [inline plugin.xml]: Are these required for Crashlytics?
I'd say that for example Firebase/Firestore is a database not related to this context

Comment (Darcelsan) [reply plugin.xml]: > Are these required for Crashlytics?
> I'd say that for example Firebase/Firestore is a database not related to this context

They are from the Firebase documentation ([https://firebase.google.com/docs/ios/setup?authuser=1](https://firebase.google.com/docs/ios/setup?authuser=1)) and they shouldn't be required but I have fatal error in ionic build command without these lines, so...

Comment (piotr-cz) [reply plugin.xml]: I see

Comment (piotr-cz) [reply plugin.xml]: Could you try with just `Firebase/Analytics` as described in https://firebase.google.com/docs/ios/setup?authuser=1#add-sdks?

There is another similar PR #30 which adds only `Firebase/Analytics` dependecy

Comment (Darcelsan) [reply plugin.xml]: > Could you try with just `Firebase/Analytics` as described in https://firebase.google.com/docs/ios/setup?authuser=1#add-sdks?
> 
> There is another similar PR #30 which adds only `Firebase/Analytics` dependecy

It works with only `Firebase/Analytics` ! I commited this change to update my PR

**Comment by rip3rs** [comment]
Any idea when this PR will be merged? :)

**Comment by rip3rs** [comment]
I can confirm that this PR will fix the issue with not sending crash analytics to firebase

**Comment by ReallySmallSoftware** [comment]
I have merged this - I have had a couple of similar changes - if I have messed up the pod specs I apologise!

**Comment by piotr-cz** [comment]
Can we have new release with this PR please?

**Comment by ReallySmallSoftware** [comment]
I haven't had time to test this on iOS, but I have published a release.

Kind regards

Richard

**Pull request merged**